mono/mini/aot-runtime.c

```diff
@@ -5167,23 +5166,32 @@ init_plt (MonoAotModule *amodule)
 	if (amodule->plt_inited)
 		return;
 
+	tramp = mono_create_specific_trampoline (amodule, MONO_TRAMPOLINE_AOT_PLT, mono_get_root_domain (), NULL);
+	tramp = mono_create_ftnptr (mono_domain_get (), tramp);
```

Comment:
so they could potentially leak, right?  I guess we don't care...

Reply:
Double checked locking introduces a lot of these small random leaks.